api/cas-server-core-api-configuration-model/src/main/java/org/apereo/cas/configuration/model/support/pac4j/oidc/Pac4jOidcClientProperties.java

```diff
@@ -43,6 +43,6 @@
      * Settings specific to delegating authentication to keycloak.
```

Comment:
Please change the comment as well: "keycloak" -> "generic"

Reply:
Comment corrected by mmoayyed